controllers/constants/constants.go

```diff
@@ -54,3 +54,4 @@ var EnvNoHTTP2 = corev1.EnvVar{
 	Name:  "GODEBUG",
 	Value: "http2server=0",
 }
+var MetricHighCardinalityLabels = []string{"DstAddr", "DstK8S_Name", "DstMac", "SrcAddr", "SrcK8S_Name", "SrcMac"}
```

Comment:
can u instead of hardcoding them to look for
https://github.com/netobserv/network-observability-operator/blob/main/controllers/consoleplugin/config/config.go#L83 with cardinalty `avoid ` similar to how I look for valueField for field of type `number` ?

Reply:
Done!